systemvm/patches/debian/config/opt/cloud/bin/merge.py

```diff
@@ -228,7 +228,7 @@ def processCLItem(self, num, nw_type):
             if('localgw' in self.qFile.data['cmd_line']):
                 dp['gateway'] = self.qFile.data['cmd_line']['localgw']
             else:
-                dp['gateway'] = 'None'
+                dp['gateway'] = self.qFile.data['cmd_line'].get('gateway', 'None')
```

Comment:
there could be a possibility of having an ipv6 gateway, in that case, we would be looking at `ip6gateway`.